Scurrets

Hobbitmax opened the pull request:
the Shit Sprite Versions


im PRETTY SURE if they merge my commits into main EE that whoever commits better sprites will merge onto this smoothly. unless some motherfucker makes a entirely new port of scurrets.


wawa.


<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

* **New Features**
  * Adds the Scurret species (visuals, body parts), new Scurret sounds/emotes and name datasets, Emotional Support Scurret role, Hydrated Scurret cargo product and themed crate contents, Scurret PDA and inventory template, new Helio Logistics poster.
  * Surveillance Camera UI updated with map-driven camera tracking.

* **Balance Changes**
  * Blood‑red hardsuit gains stamina damage resistance.
  * Emergency EVA suit supports additional wearable tag.

<sub>✏️ Tip: You can customize this high-level summary in your review settings.</sub>
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

Hobbitmax committed:
Wave 1 of building scurrets

lot of YAML copying. little additions to bits and bobs here. and otherwise getting the stage SET for the next part of getting this shit DONE

Hobbitmax committed:
Wave 2: Shitmed Boogaloo

agonizing pain. but hey it WORKS! hahhh...

Hobbitmax committed:
Wave 3: Emergency EVA Suit tweaks

Removes the blood red scurret wearability (if you can make a alternative model for it please do)

but ADDS the Emergency EVA model as its not that different... if it is. i'm not sure honestly.

Hobbitmax committed:
Wave 4: Water in scurret limbs

because its RAIN WORLD.

not funny? blame eris.

Hobbitmax committed:
Wave 5: Wawa

lets take all our new crap.
and shove it in _EE

Hobbitmax committed:
Wave 5.5: Oops missed a spot

forgot to shove the file in _shitmed into _ee lmao whoopsie

Hobbitmax committed:
Merge branch 'master' into Scurrets

TheDoctor1977 commented on Resources/Prototypes/_EE/Voice/speach_emotes.yml:
filename error 
"speach" instead of "speech"

TheDoctor1977 reviewed (commented):
nitpick

this is pretty substantial content, have you made sure it works locally?

flagging eris to do code review

VMSolidus committed:
CC-0 Harpies

VMSolidus committed:
CC-0 Harpies (#2571)

# Description

I hereby waive all copyright and related or neighboring rights together
with all associated claims and causes of action with respect to this
work to the extent possible under the law.

As the codeowner for this species, I am invoking my rights to give up my
copyright claim to the Harpy species for ss14, and release it to the
public domain under the terms of CC0. Anyone may distribute, remix,
adapt, or build upon the material in any medium or format, even for
commercial purposes.

# Changelog

:cl:
- tweak: Harpies are now public domain and may freely be ported
anywhere.

VMSolidus committed:
CC0 Space Wind

VMSolidus committed:
CC0 Space Wind V5 (#2572)

# Description

I have a simple way of proving this argument wrong. 

<img width="1080" height="907" alt="image"
src="https://github.com/user-attachments/assets/625c00c1-d358-4034-8929-21b2ba4dfa8b"
/>

I hereby waive all copyright and related or neighboring rights together
with all associated claims and causes of action with respect to this
work to the extent possible under the law.

As the codeowner for Space Wind V5, I am invoking my rights to give up
my copyright claim to all modifications I have made to the
AtmosphereSystem for ss14, and release it to the public domain under the
terms of CC0. Anyone may distribute, remix, adapt, or build upon the
material in any medium or format, even for commercial purposes.

:cl:
- tweak: Space Wind V5 is now public domain and may freely be ported
anywhere.

Pspritechologist committed:
Remove Whitelist Field SpriteComponent (#2570)

BasedUser committed:
Fix IPC Emotes (#2503)

AlgumCorrupto committed:
[Fix] Unpunchable Mouse Bug (#1922)

Co-authored-by: coderabbitai[bot] <136622811+coderabbitai[bot]@users.noreply.github.com>
Co-authored-by: DEATHB4DEFEAT <77995199+DEATHB4DEFEAT@users.noreply.github.com>

EctoplasmIsGood committed:
Buff Mechs (#2250)

Co-authored-by: Your Name <EctoplasmIsGood@users.noreply.github.com>

dootythefrooty committed:
Turret Access Group and Access Level Changes (#2268)

Co-authored-by: VMSolidus <evilexecutive@gmail.com>

mrs-teg committed:
Asterisk (#2533)

Co-authored-by: mi8hip <azizalperyk@gmail.com>

mrs-teg committed:
Hive SM Atmosfix, Cryogenics Setup (#2534)

astriloqua committed:
Better Surveillance Monitors Port (#2550)

Co-authored-by: Hagvan <22118902+Hagvan@users.noreply.github.com>
Co-authored-by: Hagvan <22118902+Hagvan@users.no.reply.github.com>

astriloqua committed:
Organise and Add Assets (#2531)

Co-authored-by: Peptide90 <User94636@protonmail.com>
Co-authored-by: FizCap <awesomekid6@live.com>
Co-authored-by: MaxxOrion <48482272+bryan-hub@users.noreply.github.com>

Hobbitmax committed:
Merge branch 'Simple-Station:master' into Scurrets
